### DISCUSSION TITLE
Using the new slice by feature does not work with iterations

### DISCUSSION THREAD

**Discussion by tincann** [community/community#67864] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

There is a bug when slicing on an iteration field.

Steps to reproduce:
- Add a couple of issues under different iterations (some in the past, some current, some in the future)
- Create a view (does not matter which type)
- Add a slice on the iteration field

When selecting completed iterations, the issues will get filtered correctly.
When selecting the current and future iterations in the slicer, the board will be empty, even though the count badges indicate that there should be issues under that iteration.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jonathanva** [comment]
Same issue here.
Slicing on iterations that have the **same name as completed iterations** gives empty boards. 

The weird thing is that filtering on @current works, only slicing on the same does not. Probably because it confuses the two with the same name.

Temp fix for us is to use 'Sprint 2025.7' unfortunately this doesn't work well with the autonumbering upon creating a new iteration.
